### ISSUE TITLE
debug包正常，release无法播放，确认不是混淆的问题...

### ISSUE THREAD

**Issue by zhuyu1022** [penfeizhou/APNG4Android#76]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by zhuyu1022** [comment]
原来用的是ResourceStreamLoader，改成AssetStreamLoader方式就可以正常播放了...

**Comment by penfeizhou** [comment]
apng资源请不要放在resource的drawable里，因为android打包在aapt可能对png类型图片进行压缩修改导致apng帧丢失，您可以选择放在raw下或者放在assets里

**Comment by penfeizhou** [comment]
这个问题关闭了，因为跟动画库本身没有直接关系，是因为release下开启了资源优化导致的。放置本地资源时注意下位置即可。

**Comment by zhuyu1022** [comment]
thanks ， 已解决问题
